package.json

```diff
@@ -19,8 +19,7 @@
     "lint:license": "node scripts/license-checker",
     "test": "lerna run build:main && lerna run test --stream --scope=$SCOPE --no-prefix",
     "bundlesize": "lerna run bundlesize --stream",
-    "serve": "node ./dev-utils/run-script.js startTestServers",
-    "karma": "karma start"
```

Comment:
removed it from here, because each package.json files already contain these and we can use npx lerna run if required.